examples/controlnet-interior-design/components/generate_prompts/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM --platform=linux/amd64 python:3.8-slim
+
+## System dependencies
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install git curl -y
+
+# Installing the package
+RUN mkdir -p /usr/local/gcloud \
```

Comment:
I think there are still some gcloud dependencies lingering here

Would remove all of 
```
# Installing the package
RUN mkdir -p /usr/local/gcloud \
&& tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
&& /usr/local/gcloud/google-cloud-sdk/install.sh

# Adding the package path to local
ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
```